<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AcceptPicturePage</class>
 <widget class="QWidget" name="AcceptPicturePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="AcceptPictureButton">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>470</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>THIS LOOKS GOOD!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RejectPictureButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>460</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>LETS TRY AGAIN!</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>791</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
